recipes/openvr/all/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup()
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
```

Comment:
```suggestion
```

I don't think it's required since the author sets -std=c++11: https://github.com/ValveSoftware/openvr/blob/v1.12.5/CMakeLists.txt#L78